### PR TITLE
Fix nctl upgrade script

### PR DESCRIPTION
**Pull request by Fraser999**
This PR fixes a recent change breaking at least one of the nightly nctl upgrade tests.

### CODE REVIEW

File: utils/nctl/sh/assets/upgrade.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ function _upgrade_node() {
     local ACTIVATE_ERA=${2}
     local NODE_ID=${3}
     local CONFIG_PATH=${4}
-    local PATH_TO_NET
+    local PATH_TO_NET=$(get_path_to_net)
```
Comment (rafal-ch) [inline utils/nctl/sh/assets/upgrade.sh:21]: In such case I think we can remove [line #29](https://github.com/casper-network/casper-node/pull/3998/files#diff-95ba73a8796b6df76ed1d8ff8a285080f0b935364ee84e134ea98e1b2123e432R29)

**Review by rafal-ch — APPROVED** [review]
LGTM :+1: 

with one nit

File: utils/nctl/sh/assets/upgrade.sh (hunk 1 above)
Comment (Fraser999) [reply utils/nctl/sh/assets/upgrade.sh:21]: Done in [13009ab](https://github.com/casper-network/casper-node/pull/3998/commits/13009ab11e1578fc165354d7956760472660fef2).

**Comment by Fraser999** [comment]
bors merge

**Pull request merged**